config/config.go

```diff
@@ -54,6 +54,8 @@ type Config struct {
 	GasPrice *big.Int
 	// InitCadenceHeight is used for initializing the database on a local emulator or a live network.
 	InitCadenceHeight uint64
+	// StartingCadenceHeight keeps track the Cadence height we start to import after restarts/deployments.
```

Comment:
I guess the chosen name for this config value is rather confusing :sweat_smile: 

To give some context of what I am trying to accomplish here:

```bash
startingBlock: QUANTITY - The block at which the import started
```

During restarts or deployments of a single Gateway instance, this is supposed to hold the Cadence height at which we started importing Flow EVM events. For the lifetime of the process, this will be the latest Cadence height saved in DB, when starting the Gateway and will not change. The `currentBlock` and `highestBlock` will continue to advance, as the Gateway continues to index new Flow EVM events, but the `startingBlock` is just a value indicating at which Cadence height we started importing, which will be different between restarts and deployments.

I am open to a better name suggestion, and moving this to the Blocks DB.

Reply:
Updated in https://github.com/onflow/flow-evm-gateway/pull/180/commits/6646a79f2aedc20774d4bad3fe43df296d73aea1